indico/modules/rb/blueprint.py

```diff
@@ -30,7 +30,7 @@
 _bp.add_url_rule('/api/<path:path>', '404', lambda path: (jsonify(), 404))
 _bp.add_url_rule('/api/config', 'config', misc.RHConfig)
 _bp.add_url_rule('/api/stats', 'stats', misc.RHStats)
-_bp.add_url_rule('/api/map-areas', 'map_areas', misc.RHMapAreas)
+_bp.add_url_rule('/api/map-areas', 'map_areas', misc.RHMapAreas, methods=('GET', 'POST', 'PATCH', 'DELETE'))
```

Comment:
~~DELETE/PATCH should have the ID in the URL (same endpoint name is fine though)~~

all the management stuff should be in `/api/admin/...` below with the other admin endpoints.

Reply:
OK, after reading the rest of the PR I see the reason for not having an ID there (updating multiple areas with a single request).

Not sure if the RESTful verbs are really appropriate for something like this. I can't say for sure which approach I'd use here but some that come to my mind:

- single POST endpoint where we simply send the new list - and then determine on the server what do delete/add/update based on whether there's an ID in the area's dict (update) or whether an areas is missing in the payload but exists on the server (delete)
- one request per area (maybe even immediately after changing something)

Reply:
Hm, I think in terms of simplicity my approach is not that bad. Those operations are nicely separated which makes it easier to understand them.